src/opower/utilities/exelon.py

```diff
@@ -25,11 +32,63 @@ def timezone() -> str:
     def login_domain() -> str:
         """Return the domain that hosts the login page."""
         raise NotImplementedError
+    
+    @classmethod
+    async def subdomain(cls) -> str:
+        """Return the opower.com subdomain for this utility."""
+        if Exelon._subdomain is not None:
+            return Exelon._subdomain
+        else:
+            # Get the potential url subdomains, they can vary based on account type/location
+            async with cls._session.get(
+                "https://" + cls.login_domain() + "/api/Services/MyAccountService.svc/GetConfiguration",

```

Comment:
I'd avoid this call and instead have a method that returns the secondary subdomain that defaults to None except for PEPCO and Delmarva.

Reply:
removed call to `GetConfiguration`